README.md

```diff
@@ -104,6 +105,25 @@ audit-ci --critical --report false
 audit-ci
 ```
 
+### Settings file
+
+**audit-ci.json**
+
+```json5
+{
+  // Only use one of ["low": true, "moderate": true, "high": true, "critical": true]
+  low: true,
```

Comment:
How about using `level: low | moderate | high`? It might deviate a little from the cli version though.

Reply:
I prefer to keep it similar to the CLI version and avoid a custom parser. I agree that the `level: "low" | "moderate" | "high" | "critical"` might look cleaner from the config settings. However, it introduces complexity from an implementation and maintenance stand-point while providing no clear benefits over the `<level>: true` alternative.